lib/boking_section.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:mobile_lab2/data.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+Widget _inputField(BuildContext context, IconData icon, String inputText) {
+  return SizedBox(
+    width: MediaQuery
+        .of(context)
+        .size
+        .width * 0.9,
+    child: TextField(
+      decoration: InputDecoration(
+          contentPadding: const EdgeInsets.all(0),
+          enabledBorder: const OutlineInputBorder(
+            borderSide: BorderSide(color: Colors.orangeAccent, width: 5.0),
+          ),
+          prefixIcon: Icon(icon),
+          hintText: inputText,
+          hintStyle: const TextStyle(fontSize: 20)),
+    ),
+  );
+}
+
+Widget _searchButton(BuildContext context) {
+  return Container(
+    width: MediaQuery
+        .of(context)
```

Comment:
no need to use multiple lines for this expression, `MediaQuery.of(context).size.width * 0.9` is fine